compiler/src/iree/compiler/Dialect/Flow/IR/test/dispatch_workgroups_folding.mlir

```diff
@@ -147,3 +147,113 @@ func.func @drop_unused_dispatch_region_result(
   // CHECK: return %[[r]]
   return %r#0 : tensor<?x?xf32>
 }
+
+// -----
+
+// CHECK-LABEL: func @bubble_up_ordinal_ops(
+func.func @bubble_up_ordinal_ops(%arg0 : index, %arg1 : index) -> tensor<?x?xf32> {
+  %result = flow.dispatch.workgroups[%arg0, %arg1](%arg0, %arg1) : (index, index) -> (tensor<?x?xf32>{%arg0, %arg1}) =
+      (%b0 : index, %b1 : index, %b2 : !flow.dispatch.tensor<writeonly:tensor<?x?xf32>>) {
+    //      CHECK: flow.dispatch.workgroups
+    // CHECK-NEXT:     %[[B0:[a-zA-Z0-9]+]]: index,
+    // CHECK-SAME:     %[[B1:[a-zA-Z0-9]+]]: index,
+    // CHECK-SAME:     %[[B2:[a-zA-Z0-9]+]]: !flow.dispatch.tensor<writeonly:tensor<?x?xf32>>
+    //  CHECK-DAG:   %[[WL1:.+]] = flow.dispatch.workload.ordinal %[[B1]] 1 : index
+    //  CHECK-DAG:   %[[WL0:.+]] = flow.dispatch.workload.ordinal %[[B0]] 0 : index
```

Comment:
it's useful for readability of they don't change ordering

Reply:
Done.